pkg/services/alerting/notifier.go

```diff
@@ -21,11 +22,11 @@ type NotifierPlugin struct {
 	Factory         NotifierFactory `json:"-"`
 }
 
-type NotificationService interface {
-	SendIfNeeded(context *EvalContext) error
-}
+// type notificationService interface {
```

Comment:
Do you want to leave it commented here?

Reply:
weird. I thought I deleted that. thanks!